project.godot

```diff
@@ -54,7 +54,6 @@ enabled=PackedStringArray("res://addons/format_on_save/plugin.cfg", "res://addon
 [rendering]
 
 textures/canvas_textures/default_texture_filter=0
-renderer/rendering_method="gl_compatibility"
```

Comment:
I'm still not sure which renderer is better in a sense to be as inclusive as possible (compatibility or forward), so I would keep the compatibility since the project is ran on the web right now, and for mac, add a profile

```
renderer/rendering_method.macos="forward_plus"
```

like suggested here: https://github.com/godotengine/godot/issues/72469#issuecomment-1932628652

until the bugfix is merged into the stable version of Godot.

Reply:
sounds good to me, I will fix the pr